org.adempiere.base/src/org/compiere/print/ReportEngine.java

```diff
@@ -2390,6 +2392,14 @@ public String getReportType()
 		return reportType;
 	}
 	
+	public boolean isOverride() {
+		return m_isOverride;
+	}
+
+	public void setIsOverride(boolean m_isOverride) {
+		this.m_isOverride = m_isOverride;
+	}
+
```

Comment:
add javadoc for public methods and I think it is more readable to use IsReplaceTabContent instead of isOverride.